Rails/Forms/odin_project_forms/test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
Missing some tests for username, email, and password presence, but I know it's not the main focus of this exercise. Just mentioning it for thoroughness's sake 😅